web/styles/app_variables.css

```diff
@@ -1,4 +1,167 @@
 :root {
+    /* Zulip color palette */
+    /* stylelint-disable color-no-hex */
```

Comment:
I thought the color palette was defined using a fancier color space; @terpimost can you clarify what we want to write here? 

(I think we can probably fix in a follow-up, given the limited use of these in this PR).